app/config/custom-environment-variables.json

```diff
@@ -9,6 +9,7 @@
     "apiPath": "FRONTEND_APIPATH",
     "basePath": "FRONTEND_BASEPATH",
     "dashboardUrl": "FRONTEND_DASHBOARDURL",
+    "inspectionDashboardUrl": "FRONTEND_INSPECTION_DASHBOARDURL",
```

Comment:
Since we're going to have multiple dashboards now, I'm going to suggest the following change:
```json
"dashboards": {
  "publicUrl": "FRONTEND_DASHBOARD_PUBLICURL",
  "inspectionUrl": "FRONTEND_DASHBOARD_INSPECTIONURL"
}
```
While this means we'd have to modify the configmap slightly for the existing variable, it logically clusters and groups similar purpose values together which will make more sense in the long run when there's multiple dashboard urls.

Reply:
➕

Reply:
sure. done